app-modules/ai/src/Filament/Pages/ManageAiSettings.php

```diff
@@ -199,7 +199,7 @@ public function getSaveFormAction(): Action
 
     public function save(): void
     {
-        if (! auth()->user()->can('product_admin.*.update')) {
+        if (! auth()->user()->can('saas_global_admin.*.update')) {
```

Comment:
This should be checking if they are a global admin rather than a permission. There is a helper for this. `$user->isSuperAdmin()`